api/v1alpha1/localvolumediscoveryresult_types.go

```diff
@@ -74,8 +48,8 @@ type LocalVolumeDiscoveryResultSpec struct {
 type LocalVolumeDiscoveryResultStatus struct {
 	// DiscoveredTimeStamp is the last timestamp when the list of discovered devices was updated
 	DiscoveredTimeStamp string `json:"discoveredTimeStamp,omitempty"`
-	// DiscoveredDevices contains the list of devices on which LSO
-	// is capable of creating LocalPVs
+	// DiscoveredDevices contains the list of devices which are usable
+	// for creating LocalDisks
```

Comment:
shouldn't we be more precise and mention that the list of devices are those that contain a WWN, perhaps? Since we are not applying the filtering criteria here. WDYT?